docs/rsdl/rapid-pro-rsdl-abnf.md

```diff
@@ -26,47 +28,43 @@ This grammar uses ABNF as defined by [RFC5234](https://tools.ietf.org/html/rfc52
 ```ABNF
 model               = 1*modelElement
 
-modelElement        = typeDefinition /
-                      serviceDefinition /
-                      enumDefinition
+modelElement        = typeDefinition
+                    / serviceDefinition
+                    / enumDefinition
 ```
 
 ### Type definition
 
 ```ABNF
-
-
 typeDefinition      = "type" "{" typeMember "}"
 
 typeMember          = property / function ; property or bound function
 
-property            = \*propertyAnnotation identifier ":" typeReference
+property            = *propertyAnnotation identifier ":" typeReference
 
-propertyAnnotation  = "@" "key"
+propertyAnnotation  = "@key"
```

Comment:
We probably don't want to allow whitespace between  `@` and `key`, same for other annotations